### DISCUSSION TITLE
export data in CSV format without creating intermediate bin files

### DISCUSSION THREAD

**Discussion by guykatz** [microstream-one/microstream#569] [Q&A] [answered]
hi all;
in the microstream docs the export to csv implies that in order to export data to CSV file format we need to go thourgh an internediate bin file.  (described in: [https://docs.microstream.one/manual/storage/import-export.html](https://docs.microstream.one/manual/storage/import-export.html)
is there a way to export to CSV withouth creating the physical bin files along the way?
thanks

**Comment by hg-ms** [comment, marked as answer]
Hello,
currently there is no way to avoid the temporary creation of the binary files. That would require a new implementation of the exporters.
